qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -187,6 +178,24 @@ def __init__(
         # Push the schedules to the instruction schedule map.
         self.update_inst_map()
 
+    @property
+    def drive_freq(self):
+        """Property only here to deprecate `drive_freq`."""
+        warnings.warn(
```

Comment:
Please use new deprecation decorator as we are under migration https://github.com/Qiskit-Extensions/qiskit-experiments/pull/1186

Reply:
1186 is not yet merged. I suggest we proceed as is in this PR and then do the switch in 1186. I have a feeling we will be able to merge 1202 first and I'd like to proceed with 1202 so that we can move forward with #1200

Reply:
Done. Thanks for the tip!